src/widgets/forms/autocomplete/language.ts

```diff
@@ -36,7 +36,7 @@ export async function getPreferredLanagugesFor (person: NamedNode) {
   const list = kb.any(person, ns.schema('knowsLanguage'), null, doc) as Collection | undefined
   if (!list) {
     console.log(`User ${person} has not set their languages in their profile.`)
-    return null // differnet from []
+    return defaultPreferedLangages
```

Comment:
Creative spellings --
* `defaultPreferedLangages` 
* `getPreferredLanagugesFor`

There may be others.  To fix the above ---

* `s/Prefered/Preferred/g`
* `s/Langages/Languages/g`
* `s/Lanaguges/Languages/g`

Reply:
Opps. global change made. Thanks!